### ISSUE TITLE
GitHub actions does not annotate files with the current formt=github output

### ISSUE THREAD

**Issue by Jeinhaus** [adrienverge/yamllint#764]
# yamllint GitHub Action annotations stopped working with file paths containing `./`

## Summary
 yamllint's `format=github` option for  GitHub Action output has stopped annotating warnings and errors on the Files tab of pull requests. The issue appears to be related to file paths that include the `./` prefix.

## Expected Behavior
yamllint errors and warnings should be annotated directly on the Files tab of pull requests, highlighting the specific lines and columns where issues occur.

## Actual Behavior
No annotations appear on the Files tab, even though yamllint is detecting and reporting errors in the workflow logs.

## Cause
GitHub workflow commands for setting messages no longer work when file paths include the leading `./` prefix. This appears to be a recent change in either yamllint output format or GitHub Actions parsing behavior.

## Reproduction Steps
1. Set up a GitHub Action with yamllint (configuration below)
2. Create a YAML file with linting errors
3. Open a pull request
4. Observe that no annotations appear on the Files tab

## Current Workflow Configuration
```yaml
---
name: Verify manifests
on:
  pull_request:
    branches:
    - main
jobs:
  yamllint:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v4
    - uses: actions/setup-python@v5
      with:
        python-version: '3.13'
    - name: Install yamllint
      run: pip install yamllint
    - name: Lint YAML files
      run: yamllint .
```

### Non-working output format (current):
```
::group::./foo/test.yaml
::error file=./foo/test.yaml,line=26,col=11::26:11 [indentation] wrong indentation: expected 12 but found 10
::endgroup::
```

### Working output format (after manual fix):
```
::group::foo/test.yaml
::error file=foo/test.yaml,line=26,col=11::26:11 [indentation] wrong indentation: expected 12 but found 10
::endgroup::
```

## Environment
- GitHub Actions runner: `ubuntu-latest`
- Python version: `3.13`
- yamllint: Latest version (installed via pip)

## Questions
- Has there been a recent change in yamllint's output format that introduced the `./` prefix?
- Is this a regression in GitHub Actions' parsing of workflow commands?
- What would be the recommended approach to fix this issue?

## Potential Solutions
1. Modify yamllint to output file paths without the `./` prefix
2. Add a post-processing step in the GitHub Action to strip the `./` prefix
